keras/engine/training.py

```diff
@@ -321,6 +322,36 @@ def __setattr__(self, name, value):
 
     super(Model, self).__setattr__(name, value)
 
+  def __reduce__(self):
+    if self.built:
+      return (deserialize_model_from_bytecode, serialize_model_as_bytecode(self))
+    else:
+      # SavedModel (and hence serialize_model_as_bytecode) only support built models,
+      # but if the model is not built,
+      # it _may_ be possible to serialize as a plain Python object,
+      # as long as the constituent parts (layers, optimizers, losses, etc.)
+      # can be serialized as plain Python objects.
+      # Thus we call up the MRO to get an implementation of __reduce__
```

Comment:
MRO?

Reply:
MRO == Method Resolution Order here, but I admit it's probably not the best term to use, much less abbreviated.

How about `Thus we call up the superclass hierarchy to get an implementation of __reduce__`?